mk/uvmt/dsim.mk

```diff
@@ -26,38 +26,51 @@
 DSIM                    = dsim
 DSIM_HOME              ?= /tools/Metrics/dsim
 DSIM_CMP_FLAGS         ?= $(TIMESCALE) $(SV_CMP_FLAGS) -top uvmt_$(CV_CORE_LC)_tb
-DSIM_ERR_SUPPRESS      ?= MultiBlockWrite:ReadingOutputModport
+DSIM_ERR_SUPPRESS      ?= MultiBlockWrite:ReadingOutputModport:IneffectiveDynamicCast:IndexOOB:PortWidthMismatch:UninstVif:LatchInferred:AlwaysFFNba
```

Comment:
The list of additional errors to suppress calls for a thorough review+fix of the RTL...

Reply:
Yes, DSim is very, very fussy.  Once this is merged in, I will create a set of Issues related to each of these suppressions.